src/components/Radio/Radio.test.tsx

```diff
@@ -0,0 +1,123 @@
+import { expect } from "chai";
+import * as Enzyme from "enzyme";
+import * as React from "react";
+import { stub } from "sinon";
+
+import Radio from "./Radio";
+
+describe("Radio Button", () => {
+  let container;
+  let changeHandler;
+  let clickHander;
+
+  before(() => {
+    clickHander = stub();
+    changeHandler = stub();
+    container = Enzyme.mount(
+      <Radio
+        id="inputID"
+        attributes={{ onClick: clickHander }}
+        onChange={changeHandler}
+        labelText="Test Label"
+        showLabel={false}
+      ></Radio>
+    );
+  });
+
+  it("Renders with appropriate 'aria-label' attribute and value when 'showLabel' prop is set to false", () => {
+    const labelText = container.prop("labelText");
+    expect(container.find("input").prop("aria-label")).to.equal(labelText);
+  });
+
+  it("Renders without crashing", () => {
+    expect(container.find("input").exists()).to.equal(true);
+  });
+
+  it("The radio element is an input with type='radio'", () => {
+    expect(container.find("input").prop("type")).to.equal("radio");
+  });
+
+  it("Allows user to pass in additional attributes", () => {
+    container.simulate("click");
+    expect(clickHander.callCount).to.equal(1);
+  });
+
+  it("Changing the value calls the onChange handler", () => {
+    container.find("input").simulate("change", { target: { value: "Hello" } });
+    expect(changeHandler.callCount).to.equal(1);
+  });
+
+  it("Renders with label", () => {
+    container = Enzyme.mount(
+      <>
```

Comment:
Nit: you don't need the fragment element wrapper.

Reply:
Good catch; fixed!

Reply:
Fixed!